app/main/services/response_formatters.py

```diff
@@ -86,13 +86,14 @@ def api_response(data, status_code, key='message'):
 
     See http://elasticsearch-py.readthedocs.io/en/master/exceptions.html#elasticsearch.TransportError.status_code for
     an explaination of 'N/A' status code. elasticsearch-py client returns 'N/A' as status code if ES server cannot be
-    reached
+    reached, which is caught by `except TypeError` below.
+    It's possible that ElasticSearch can also return other unexpected non-integer status codes, which will also get
```

Comment:
We should really say "it's possible for the ElasticSearch library to return [etc]" - because it really isn't possible to get a non-integer status code over HTTP :)

Reply:
Cool, have changed.